source/backup-cursor.rst

```diff
@@ -0,0 +1,95 @@
+.. _backup-cursor:
+
+============================================================
+$backupCursor and $backupCursorExtended aggregation stages
+============================================================
+
+``$backupCursor`` and ``$backupCursorExtended`` aggregation stages allow developing your custom backup solution: 
```

Comment:
it's valid use-case but IMHO not the feature description) Maybe add smth like - New aggregation pipeline stages expose WiredTiger API which gives a way to make consistent backup by listing all files that should be copied for backup and freezing those files so you can copy them without them being deleted or necessary parts within them being overwritten - just compilation of Akira's thoughts:)

Reply:
I like this suggestion